.travis.yml

```diff
@@ -23,7 +23,7 @@ branches:
     - master
 
 before_install:
-  - nvm install --latest-npm
+  - nvm install
```

Comment:
Not sure we should be changing this

Reply:
This seems to be a change that is already on master.
0b4d9c805fb764c1ba05ebe999c76e2bf0452be6

I'm not sure why is showing up here, but we should probably update from master after we tag the last commit for the release.

Reply:
It's showing up because of this: https://github.com/WordPress/gutenberg/pull/17181/commits/1ca5e2b2f9d48f241ab3943848e27e20901eac3e

Not really sure what happened there 🤔